src/mame/bfm/bfm_gu96x8m_k657c2.cpp

```diff
@@ -0,0 +1,1010 @@
+// license:BSD-3-Clause
+// copyright-holders:
```

Comment:
We do need this to list something for the copyright holder :-)

Reply:
OK, I can put my name there.
As this file was derived from bfm_bda.cpp by James Wallace do I need to do something to acknowledge that ?